### ISSUE TITLE
Incorrect behaviour of backward keyboard navigation in Treemap and Heatmap

### ISSUE THREAD

**Issue by franklinefrancis** [highcharts/highcharts#15416]
#### Expected behaviour
Backward navigation using keyboard in Treemap and Heatmap should work similar to other charts

#### Actual behaviour
Backward navigation using Shift+Tab keys within Treemap and Heatmap needs to be done twice to move focus out of the chart.

1. Run the JSFiddle for Treemap or Heatmap
2. Start focus with the first text field labelled 'First focus'
3. Press Tab key to move focus to chart (works as expected)
4. Press Tab key again to move focus to text field labelled 'Last focus' (works as expected)
5. Now, press Shift+Tab from 'Last focus' text field. Focus moves back to chart data point as expected.
6. Press Shift+Tab to move focus back to 'First focus' text field. But focus moves to another data point within chart. Only upon pressing Shift+Tab again, the focus moves back to 'First focus' text field. Thus backward keyboard navigation order is different from forward navigation.

![Heatmap Backward Keyboard Navigation](https://user-images.githubusercontent.com/1227574/112713343-9e13d480-8efa-11eb-82e0-58bfb43d4d53.PNG)

![Treemap Backward Keyboard Navigation](https://user-images.githubusercontent.com/1227574/112713797-ed5b0480-8efc-11eb-91ba-d8c2d2d9e640.PNG)

#### Live demo with steps to reproduce
Treemap - https://jsfiddle.net/franklinefrancis/yaj849uo/
Heatmap - https://jsfiddle.net/franklinefrancis/4wj9efdg/

#### Product version
8.2.0

#### Affected browser(s)
Chrome, Firefox

**Comment by KacperMadej** [comment]
Thank you for reporting the bug.

fyi: a11y @oysteinmoseng

**Comment by franklinefrancis** [comment]
@oysteinmoseng, thanks for fix this. Can this be backported to 8.2.0?

**Comment by oysteinmoseng** [comment]
@franklinefrancis Unfortunately this fix is not easily backportable since it required core refactoring. Are you unable to update to the latest version?

**Comment by franklinefrancis** [comment]
@oysteinmoseng, we cannot update to latest version since we have a wrapper API based on 8.2.0.

**Comment by oysteinmoseng** [comment]
@franklinefrancis The current version is backwards compatible with 8.2.0, with the only compatibility note being related to HTML content security. See [changelog](https://www.highcharts.com/blog/changelog/). Any compatibility issues should be easily resolved - and of course we are happy to assist if needed.
